app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -20,6 +20,9 @@ def textual_group_smart_management
     %i(zone tags)
   end
 
+  def textual_group_topology
```

Comment:
they all use the same common ems view, so this should be better than having conditions in a view